README.md

```diff
@@ -66,6 +66,13 @@ Simulate a crash (for testing purpose)
 
 ### `throwException()`
 
+Used with EventListener "uncaughtException" function(evt)
+
+example:             Crashlytics.throwException(evt.message,evt.line,evt.sourceName,evt.lineSource,evt.javascriptStack.toString());
+
+
+### `throwCustomException(String message, String myline, String sourceNameAndroid, String lineSource, String javascriptStack)`
+
```

Comment:
The example is linked to the `throwException` method now, not the `throwCustomException`.

Reply:
sorry about that. 
Fixed the readme

Reply:
it still says `throwException` in the example (line 76)